### DISCUSSION TITLE
Smart Garage Controller WG088 model

### DISCUSSION THREAD

**Discussion by Juananor3** [milo526/homebridge-tuya-web#260] [General]
Hello good morning, I recently bought a Smart Garage Controller specifically the WG088 model, in principle I have it installed in your application without any problem and working perfectly but this device is not recognized in Homebridge while other devices of the same account have. Tried several times but I still can't find the solution.
You will know tell me a solution or what the problem is or if I need another plugins in Homebridge.

Greetings and thanks

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.
